### ISSUE TITLE
Enhancement: MATSIM plug-in (moved from gama-issues)

### ISSUE THREAD

**Issue by AlexisDrogoul** [gama-platform/gama.experimental#4]
Describe which enhancement/addition to GAMA you would like to see:

MATSIM is a well-known open-source traffic simulation framework (http://www.matsim.org).
It is developed in JAVA. 
It could be nice to have a MATSIM plug-in in GAMA enabling to easily use MATSIM features
through the GAML language. 

(moved from https://github.com/gama-platform/gama/issues/1033)

**Comment by agrignard** [comment]
Any news on this coupling? I have also some demand on using this framework https://its.mit.edu/research/simmobility I don't how easy/difficult it can be to couple this to GAMA...

**Comment by ptaillandier** [comment]
No.... But I think it could be a good Master student internship subject ;) .


2016-09-27 20:58 GMT+02:00 Arnaud Grignard <notifications@github.com>:
